networkpolicy/README.md

```diff
@@ -0,0 +1,81 @@
+# Overview
+
+This folder contains the list of network policies those required for Bedrock Common Services to work in a cluster where the deny_all policy has been implemented. The script **install_networkpolicy.sh** provided to install all the YAML definition of NetworkPolicies for Bedrock Common Services to the specified namespace. If you have the deny-all policy in place, the ingress traffic to all pods in not allowed. In this scenario, for foundational services to work, you need to import and install network policies. If you do not use deny-all policy, you may not need to import or install network policies.
+
+For more details on the usage of the script and to check various supported inputs , refer to the [document section](https://www.ibm.com/docs/en/cpfs?topic=operator-installing-network-policies)
+
+## Sample usage - installing Network Policies
+
+```
+networkpolicy % ./install_networkpolicy.sh   
+# [0] Checking prerequesites ...
+-----------------------------------------------------------------------
+[✔] oc command available
+[✔] oc command logged in as [redacted]
+[✔] IBM Common Services found in namespace ibm-common-services
+# [0] Installing Bedrock Network Policies ...
+-----------------------------------------------------------------------
+ibm-common-services
+ibm-common-services
+[INFO] Installing 00-default-deny-all.yaml ...
+networkpolicy.networking.k8s.io/default-deny-ingress created
```

Comment:
@ujjwalchk-it Should we should the deny-all information from the Readme as well?